apps/examples.nameguard.io/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "nextjs-sdk",
+  "name": "nameguard-examples",
```

Comment:
This is an example, we can get rid of this and version.

Reply:
This is useful during the build process.

![CleanShot 2024-05-16 at 16 12 57](https://github.com/namehash/nameguard/assets/126201998/deeb9931-ecaf-4838-a418-5be6af40a872)

Reply:
Removed version :+1: